library/math.sol

```diff
@@ -8,4 +9,167 @@ library Math {
             x = mulmod(x, x, m);
         }
     }
+    
+    /// @dev unsigned constant infinity (largest number possible)
+    function Infinity() constant returns (uint inf) {
```

Comment:
This does not really behave as "infinity". What about `uint_max`? (also applies to the others)